backend/api/Mapping/TypeMap.cs

```diff
@@ -14,6 +14,12 @@ public void Register(TypeAdapterConfig config)
                 .Map("IsDisabled", "IsDisabled")
                 .Map("DisplayOrder", "DisplayOrder");
 
+            config.ForType(typeof(Entity.ICodeEntity<short>), typeof(Model.TypeModel<short>))
+                .Map("Id", "Id")
+                .Map("Description", "Description")
+                .Map("IsDisabled", "IsDisabled")
+                .Map("DisplayOrder", "DisplayOrder");
+
```

Comment:
I like this generic mapping but ICodeEntity doesn't have Id on the interface (instead has Code). 
I see that of the two code entities below (Region and District) one has Id and the other hasn't.
What do you think of adding an "Id" property to the ICodeEntity interface to make it consistent across all models? 
Otherwise this mapping would fail for some code entities I believe? 
Thoughts?

Reply:
This was created to handle some exceptions to the normal structure of typed entities, I think Doug created these early in the project before he had created an established pattern.

So for these ones I think mapping from the Id to the code makes sense (as the code is the only identifier on these tables).